api/app/utils/hfi_calculator.py

```diff
@@ -6,6 +6,45 @@
 from app.db.database import get_read_session_scope
 
 
+# PC, PDF, CC, CDH from the Red Book. Assumes values of 1 CBH.
```

Comment:
An old assumption of CBH by yours truly

Reply:
Added this comment and others to tech. debt ticket.